src/utils/regex.js

```diff
@@ -1,22 +1,18 @@
 let _ = require('lodash/fp')
 
-let toSafeRegex = _.flow(
-  _.replace(/[.?+*|{}[\]()]/g, '\\$&'),
-  _.map(x => (x.match(/[A-Za-z]/) ? `[${_.toUpper(x)}${_.toLower(x)}]` : x)),
-  _.join('')
-)
+let toSafeRegex = _.flow(_.replace(/[.?+*|{}[\]()]/g, ''))
```

Comment:
We should replace these with spaces to more closely match how es will analyze these. If you search for `one/two`, you’d want it to run `one two` and not `onetwo`